src/bin/miri.rs

```diff
@@ -14,6 +14,7 @@ extern crate rustc_codegen_utils;
 extern crate rustc_interface;
 extern crate syntax;
 
+use hex::FromHexError;
```

Comment:
Please keep `std` imports separately. This new import should be below the `rustc` imports, in fact.

Reply:
Does my new change look right?